projects/scion/microfrontend-platform/src/lib/host/message-broker/topic-subscription.registry.spec.ts

```diff
@@ -455,9 +455,9 @@ describe('TopicSubscriptionRegistry', () => {
     });
   });
 
-  function expectSubscriptionCount(topic: string): { toBe: (expected: any) => Promise<void> } {
+  function expectSubscriptionCount(topic: string): { toBe: (expected: any) => PromiseLike<void> } {
```

Comment:
Why do we need `PromiseLike`?

Reply:
The method `toBeResolvedTo` on line 461 returns a `PromiseLike`, the return type of this function has to be the same to avoid TS errors.

Reply:
Changed to async function with Promise